src/failures/src/communication/readers/queued-sim-var-reader.test.ts

```diff
@@ -0,0 +1,53 @@
+import { QueuedSimVarReader } from '.';
+import { SimVarReaderWriter } from '..';
+
+test('does not read a value that is not in the collection', async () => {
+    await expectToBeCalledTimes(0, async (r) => {
+        await SimVar.SetSimVarValue(simVarName, 'number', notInCollectionIdentifier);
+        r.update();
+    });
+});
+
+test('reads a value that is in the collection', async () => {
+    await expectToBeCalledTimes(1, async (r) => {
+        await SimVar.SetSimVarValue(simVarName, 'number', inCollectionIdentifier);
+        r.update();
+    });
+});
+
+test('doesn\'t read the same value multiple times', async () => {
```

Comment:
Personally I fall back from single quotes to double quotes for string containing a single quote, but that's a preference:

```suggestion
test("doesn't read the same value multiple times", async () => {
```

I believe prettier would do this as well, if we ever used it.

Reply:
Agreed. I believe somehow our eslint rules conditioned me into thinking that was not an option, but it is also my preferred method for handling such strings. 😀

Reply:
We can enable the `avoidEscape` option to allow this

Reply:
Nothing needs to be changed to allow this. It's already allowed.

Reply:
Ah okay, I thought you were implying that eslint was complaining.

Reply:
What I was trying to say is that eslint complains so much that by now I even believe eslint complains when it doesn't! 😀